tests/Basic/Basic.Tests.cs

```diff
@@ -522,5 +522,17 @@ public void TestMarshallingEmptyType()
     {
         var empty = new ReturnsEmpty().Empty;
     }
+
+    [Test]
+    public void TestClassesPassTry()
+    {
+        RefTypeClassPassTry refTypeClassPassTry;
```

Comment:
You're still not asserting anything. We should have some fields and make sure they're being marshaled correctly.